### ISSUE TITLE
Figure 9 - stage 4 improvment?

### ISSUE THREAD

**Issue by Mikey-E** [TangYuan96/MiniGPT-3D#9]
In the paper figure 9, it looks like the stage 4 training (red - training the query experts) has not improved the loss compared to stage 3 (green). Can you confirm whether or not it helped in any way?

**Comment by TangYuan96** [comment]
Thank you for your interest in our work.

As shown in Figure 9,  the red line (Stage IV) exhibits a slight decrease in loss compared to the green line （Stage III）. 

In Stage IV, we only fine-tuned 0.4M parameters of the MQE module, where multiple query experts are leveraged to extract and aggregate information from point tokens through diverse feature perspectives, yet this significantly enhances the LLM's 3D information ingestion capability.

 From the ablation study in Table 5 (rows 3 vs. 4), we observe that adding Stage IV leads to a notable performance gain (+1.46 points）. The model weights for both Stage III and Stage IV have been open-sourced.
